src/app/remotes/change-password/change-password.component.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  DialogState,
+  PortalCoreModule,
+  PortalDialogService,
+  PortalMessageService,
+  UserService,
+  createRemoteComponentTranslateLoader,
+  providePortalDialogService
+} from '@onecx/portal-integration-angular'
+import { Observable, ReplaySubject, mergeMap, of } from 'rxjs'
+import { RippleModule } from 'primeng/ripple'
+import { TooltipModule } from 'primeng/tooltip'
+import { PrimeIcons } from 'primeng/api'
+import { SharedModule } from 'src/app/shared/shared.module'
+import { ChangePasswordDialogComponent } from './change-password-dialog/change-password-dialog.component'
+// import { IAMAPIService, ResetPasswordRequestParams, UserResetPasswordRequest } from 'src/app/shared/generated'
+// import { UserProfileService } from '../user-profile.service'
+
+@Component({
+  selector: 'app-ocx-change-password',
+  templateUrl: './change-password.component.html',
+  styleUrls: ['./change-password.component.scss'],
+  standalone: true,
+  imports: [
+    CommonModule,
+    HttpClientModule,
+    RippleModule,
+    TooltipModule,
+    ChangePasswordDialogComponent,
+    TranslateModule,
+    SharedModule,
+    PortalCoreModule,
+    AngularRemoteComponentsModule
+  ],
+  providers: [
+    // api
+    PortalMessageService,
+    providePortalDialogService(),
+    {
+      provide: BASE_URL,
+      useValue: new ReplaySubject<string>(1)
+    },
+    provideTranslateServiceForRoot({
+      isolate: true,
+      loader: {
+        provide: TranslateLoader,
+        useFactory: createRemoteComponentTranslateLoader,
+        deps: [HttpClient, BASE_URL]
+      }
+    })
+  ]
+})
+export class OneCXChangePasswordComponent implements ocxRemoteComponent {
+  // permissions: string[] = []
+  permissions: string[] = ['PROFILE_PASSWORD#EDIT']
+
+  constructor(
+    @Inject(BASE_URL) private baseUrl: ReplaySubject<string>,
+    // private readonly userProfileService: UserProfileService,
+    // private readonly iamService: IAMAPIService,
+    private userService: UserService,
+    private portalDialogService: PortalDialogService,
+    private msgService: PortalMessageService,
+    private translateService: TranslateService
+  ) {
+    this.userService.lang$.subscribe((lang) => this.translateService.use(lang))
+  }
+
+  ocxInitRemoteComponent(config: RemoteComponentConfig): void {
+    this.baseUrl.next(config.baseUrl)
+    // this.permissions = config.permissions
+    // this.helpDataService.configuration = new Configuration({ //   basePath: Location.joinWithSlash(config.baseUrl, environment.apiPrefix) // })
+  }
+
+  private openChangePasswordEditorDialog(): Observable<DialogState<string>> {
+    return this.portalDialogService.openDialog<string>(
+      'CHANGE_PASSWORD.DIALOG.TITLE',
+      {
+        type: ChangePasswordDialogComponent,
+        inputs: {}
+      },
+      {
+        key: 'CHANGE_PASSWORD.DIALOG.CHANGE_BUTTON'
+      },
+      {
+        key: 'CHANGE_PASSWORD.DIALOG.CANCEL'
+      }
+    )
+  }
```

Comment:
```
this.portalDialogService.openDialog<string>(
      'CHANGE_PASSWORD.DIALOG.TITLE',
      ChangePasswordDialogComponent,
      'CHANGE_PASSWORD.DIALOG.CHANGE_BUTTON',
      'CHANGE_PASSWORD.DIALOG.CANCEL'
    )
  }
```
should work

Reply:
Is it not working that way?

Reply:
It is, for some reason I missed button changes :/